src/tests/AccountCreation.test.js

```diff
@@ -0,0 +1,35 @@
+import '@testing-library/jest-dom/extend-expect'
+import { fireEvent, render, waitFor } from '@testing-library/react';
+import { postCheckingAccounts } from '../api';
+import {CheckingAccountCreation} from '../components/CheckingAccountCreation'
+
+//this test is not finished yet
+jest.mock('../api/index.js');
+
+test('Should create the checking account with the given info',async()=>{
+
+    const response = {
+      status:200
+    }
```

Comment:
You should place the constant outside the `test` block.

Reply:
Yes, that's right